esp/mlrs-wifi-bridge/mlrs-wifi-bridge-boards.h

```diff
@@ -215,6 +228,44 @@ GPIO15 = RTC_GPIO13
     #define SERIAL_INVERT true
 
 
+//-- M5Stack M5Stamp C3U Mate
+#elif defined MODULE_M5STAMP_C3U_MATE // ARDUINO_ESP32C3_DEV, ARDUINO_BOARD == ESP32C3_DEV
+    #ifndef ARDUINO_ESP32C3_DEV // ARDUINO_BOARD != ESP32C3_DEV
+	      #error Select board ESP32C3_DEV!
+    #endif
+
+    #undef USE_SERIAL_DBG1
+    #define USE_SERIAL1_DBG
+
+    #undef LED_IO
+    #define USE_LED
+    #include <Adafruit_NeoPixel.h>
+    #define NUMPIXELS  1
+    Adafruit_NeoPixel pixels(NUMPIXELS, 2, NEO_GRB + NEO_KHZ800);
```

Comment:
I'm a bit confused. We have 4 boards now with NeoPixels, yet the code looks quite different each time. Two have PIN_NEOPIXEL defined, two not. Two use #ifdef NEOPIXEL_POWER, two not. What is the pattern here? Can this somehow be streamlined? I understand it's nfc, but it just doesn't look nice and systematic.
My guts feeling is that:
- here we also want to define/use PIN_NEOPIXEL, but do not want to add the #ifdef NEOPIXEL_POWER stuff
- for the M5 STAMP PICO we want to remove the #ifdef NEOPIXEL_POWER stuff
- for the M5Stamp C3 Mate we want to use PIN_NEOPIXEL

Reply:
I've just been following the example of a similar board, so these inconsistencies were already present.

But I'm also confused.
I don't see anything that ever defines  NEOPIXEL_POWER.  Is any of that code needed at all?
I don't see anything that defines PIN_NEOPIXEL when MODULE_ADAFRUIT_QT_PY_ESP32_S2 is defined.  Does this board compile?

Reply:
The "ADAFRUIT_QT_PY_ESP32_S2" has a dedicated pin to be able to switch the WS2812 on an off for IoT Low Power applications. This feature may be not available on the M5-Stamp-C3.

[https://learn.adafruit.com/adafruit-qt-py-esp32-s2/pinouts#neopixel-led-3107926](url)

Reply:
yes, the PyQt sure does compile (and the power stuff is needed)(and enclosing it in the if comes from an example of Adafruit)

Reply:
based on my version for one of the M5's I suspect that the power stuff isn't needed for all three m5's, hence the suggestion to remove it there. Defining PIN_NEOPIXEL looks nice to me as it tells what the number means, hence my suggestioon to do that for all three.

Reply:
if in this PR or a follow up PR I don't mind, I think it would be just easier to grab this occasion
I can't test the two newly added M5's, so, preferably you do it, either here or in a follow up, otherwise I would need explict confirmation that the suggested changes are ok so I could do the follow up PR.